### DISCUSSION TITLE
Is there a way to find out why the cache missed?

### DISCUSSION THREAD

**Discussion by Dragon-Hatcher** [urql-graphql/urql#2526] [Q&A] [answered]
I am aware that you can figure out if the cache was used by looking at the operation context `operation.context.meta.cacheOutcome` but I was wondering if there is a way to find out exactly what fields missed. I think that my data should be available completly from the cache but it isn't and I want to know why.

Thanks!

**Comment by JoviDeCroock** [comment, marked as answer]
The urql devtools might be able to help you here, they visualize what's cached/.... https://github.com/FormidableLabs/urql-devtools
